### ISSUE TITLE
"Error: invalid_grant" when login with Google

### ISSUE THREAD

**Issue by carvalhorafael** [outline/outline#1245]
I investigated more and found what I believe to be the mistake I mentioned here https://github.com/outline/outline/issues/1243

After authenticating with Google account (when returning to the application), the following error occurs:

```
Error: invalid_grant
  File "/srv/www/outline/releases/20200426125042/node_modules/gaxios/src/gaxios.ts", line 109, col 15, in Gaxios._request
  ?, in runMicrotasks
  File "internal/process/task_queues.js", line 97, col 5, in processTicksAndRejections
  File "/srv/www/outline/releases/20200426125042/node_modules/google-auth-library/build/src/auth/oauth2client.js", line 122, col 21, in OAuth2Client.getTokenAsync
    const res = await this.transporter.request({
```
![image](https://user-images.githubusercontent.com/496442/80309210-f18d1800-87a9-11ea-8fa7-391c5d9e5fd6.png)

**Comment by tommoor** [comment]
There's no known problem with the software here I'm afraid, this login works on getoutline.com and for hundreds of other self-installs, here's a blog post with 11 reasons that Google might send this error:
https://blog.timekit.io/google-oauth-invalid-grant-nightmare-and-how-to-fix-it-9f4efaf1da35

**Comment by carvalhorafael** [comment]
Thanks @tommoor 

I am still not sure the reason of this error. But I managed to solve it. I'll leave here what I did.

I was trying to `start a new application connected to another application's database`.

After think a little bit I tried another approach:

- I backed up the database of the old application
- started a new application with a new database
- restored the backup, and 
- ran the migrations.

After doing it, everything worked well.
